.github/workflows/create-release.yaml

```diff
@@ -27,12 +27,26 @@ jobs:
           git config user.email noreply@github.com
       - name: Pushing tags
         run: |
+          VER=$(echo v${{ env.RELEASE_VERSION }} | sed 's/.*rc.*/rc/')
+          echo "RC=$VER" >> $GITHUB_ENV
           git tag -a -m "Release v${{ env.RELEASE_VERSION }}" "v${{ env.RELEASE_VERSION }}"
           git push origin v${{ env.RELEASE_VERSION }}
       - name: Extract release notes
         id: extract-release-notes
         uses: ffurrer2/extract-release-notes@v1
+      - name: Create pre-release
+        if: env.RC == 'rc'
+        uses: actions/create-release@v1
+        with:
+          tag_name: v${{ env.RELEASE_VERSION }}
+          release_name: Release v${{ env.RELEASE_VERSION }}
+          draft: false
+          prerelease: true
```

Comment:
instead of having two stepos would it be possible to do something like:
`prerelease: ${{ env.RC == 'rc' }}`

that'd make the task cleaner looking.

Reply:
Ohh, is that boolean? I'll try that and let you know

Reply:
It works, thank you!